packages/hub/src/wallet/db/migrations/20191111110900_allocations.ts

```diff
@@ -1,7 +1,7 @@
 import * as Knex from 'knex';
 import {addBytesCheck} from '../utils';
 
-const TABLE_NAME = 'allocations';
```

Comment:
@snario in theory, an existing migration should never change. If an existing migration changes, every database has to be wiped and migrated from empty state.